rundeckapp/grails-app/services/rundeck/services/ProjectService.groovy

```diff
@@ -1947,6 +1954,43 @@ class ProjectService implements InitializingBean, ExecutionFileProducer, EventPu
         return result
     }
 
+    Boolean createAsyncImportStatusFile(String projectName){
+        try{
+            return asyncImportService.createStatusFile(projectName)
+        }catch(Exception e){
+            e.printStackTrace()
+            throw e
+        }
+    }
+
+    AsyncImportStatusDTO getAsyncImportStatusFileForProject(String projectName){
+        try{
+            def dto = asyncImportService.getAsyncImportStatusForProject(projectName)
+            if( !dto || null == dto ) throw new Exception("Status object empty or non-existent")
+            return dto
+        }catch(Exception e){
+            e.printStackTrace()
+        }
+    }
+
+    void beginAsyncImportMilestone(
```

Comment:
requires javadoc

Reply:
Added additional tests to assert exception